library/pk_wrap.c

```diff
@@ -1017,6 +1017,7 @@ static int ecdsa_sign_wrap(mbedtls_pk_context *pk, mbedtls_md_type_t md_alg,
 #endif /* MBEDTLS_USE_PSA_CRYPTO */
 #endif /* MBEDTLS_PK_CAN_ECDSA_SIGN */
 
+#if defined(MBEDTLS_ECP_LIGHT)
```

Comment:
Hmm, that feels wrong. This is already guarded by `ECDSA_C` which already depends on `ECP_C` which is a superset of `ECP_LIGHT`, so we shouldn't be needing this. I anything going wrong without it?

Reply:
My mistake, sorry. At the beginning the `ECP_LIGHT` was embracing all the ECDSA code, but then the outer guard was replaced with `MBEDTLS_PK_HAVE_ECC_KEYS`, so I thought that keeping the `ECP_LIGHT` part could be useful. However I didn't notice the `ECDSA_C` guard there.